examples/stars/main.go

```diff
@@ -0,0 +1,129 @@
+// Copyright 2021 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build example
+
+package main
+
+import (
+	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/ebitenutil"
+	"image/color"
+	"log"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+const (
+	screenWidth  = 640
+	screenHeight = 480
+
+	frameOX     = 0
+	frameOY     = 32
+	frameWidth  = 32
+	frameHeight = 32
+	frameNum    = 8
+	scale       = 64
+	points      = 1024
+)
+
+func abs(a int) int {
+	if a < 0 {
+		return -a
+	}
+	return a
+}
+
+type Star struct {
+	fromx, fromy, tox, toy, blightness float64
```

Comment:
Should this be `brightness` instead of `blightness`?

Reply:
done.